Kristen-Hu/Kristen-Hu.md

```diff
@@ -0,0 +1,14 @@
+---
+id: "05daa2a9-8768-446e-b9d3-580c8a6f9b8b"
```

Comment:
Your userid is incorrect. It should be 31aa42cd-719c-e911-825f-00155d01ef64

Reply:
Done.